acme_diags/driver/utils/general.py

```diff
@@ -11,6 +11,20 @@
 from acme_diags import container
 from acme_diags.derivations.default_regions import regions_specs
 
+def strictly_increasing(L):
+    return all(x<y for x, y in zip(L, L[1:]))
+
+def strictly_decreasing(L):
+    return all(x>y for x, y in zip(L, L[1:]))
+
+def non_increasing(L):
```

Comment:
It might be better to call this function `monotonically_decreasing` and the next one `monotonically_increasing`; that way we aren't naming things in the negative.

Reply:
okay